rskj-core/src/test/java/co/rsk/peg/PegTestUtils.java

```diff
@@ -92,6 +94,12 @@ public static Sha256Hash createHash(int nHash) {
         return Sha256Hash.wrap(bytes);
     }
 
+    public static Script createBaseInputScriptThatSpendsFromTheRedeemScript(Script redeemScript) {
```

Comment:
There a method here `createBaseRedeemScriptThatSpendsFromTheFederation` that seems like it's not used by anyone. And doesn't seem to make much sense either. Let's remove it

Reply:
is being used in some tests, as there are some where we do not have the federation, but the redeem script

Reply:
I mean another mehtod in this class `createBaseRedeemScriptThatSpendsFromTheFederation`. That one is not used anywhere

Reply:
oh ok!